### PR TITLE
Fix watch asset

### PR DESCRIPTION
**Pull request by brigittaforrai**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Building with commit 3c736776d93fa7eccf99ec124da6af107bc16ac2

https://app.netlify.com/sites/augmint/deploys/601191617494cf0008145a20

**Review by gyurka00 — COMMENTED** [review]
Why do you remove the yarn lock?

**Comment by brigittaforrai** [comment]
ahh.. coincidence

**Review by gyurka00 — APPROVED** [review]

**Review by szerintedmi — DISMISSED** [review]

File: src/components/watchAssetButton.js [hunk 1]
```diff
@@ -72,7 +72,8 @@ export class WatchAssetButton extends React.Component {
                 account: web3.userAccount
             };
 
-            this.isMetamask = window.ethereum.isMetamask;
+            const eth = window.ethereum;
+            this.isMetamask = eth ? eth.isMetamask : null;
```
Comment (gyurka00) [inline src/components/watchAssetButton.js:76]: I think you can use `window.ethereum?.isMetamask || null`

Comment (gyurka00) [reply src/components/watchAssetButton.js:76]: `this.isMetamask = window.ethereum?.isMetamask || null;`

**Review by treerz — DISMISSED** [review]
just deploy it

the whole website is broken without this fix, it cannot get worse than that

**Review by gyurka00 — APPROVED** [review]

**Review by gyurka00 — APPROVED** [review]

**Review by treerz — APPROVED** [review]

**Pull request merged**